tests/library/modernizr.spec.ts

```diff
@@ -41,6 +41,9 @@ it('safari-14-1', async ({ browser, browserName, platform, server, headless, isM
 
   if (platform === 'linux') {
     expected.subpixelfont = false;
+    expected.applicationcache = false;
```

Comment:
```
    expected.applicationcache = false;
    expected.inputsearchevent = false;
```
is already the case on mac as well. Does Windows differ or we can align across all OSes?